src/mixins/legal-api-mixin.ts

```diff
@@ -166,6 +166,16 @@ export default class LegalApiMixin extends Vue {
       })
   }
 
+  /**
+   * Fetches conversion addresses.
+   * @returns a promise to return the data
+   */
+  async fetchConversionAddresses (): Promise<any> {
+    if (!this.getBusinessId) throw new Error('Invalid business id')
+    const url = `businesses/${this.getBusinessId}/addresses`
+    return axios.get(url)
+  }
```

Comment:
I see nothing conversion-specific in here. It looks like `fetchAddresses()` but with fewer checks.

I think this method is not needed with different code to fetch the conversion snapshot (see my other comment).

Reply:
`fetchAddresses()` will throw error if axios receives errors, so I had to use another function.

Reply:
You can catch the error in Conversion.vue and handle it appropriately there.